packages/react-components/src/components/CameraButton.tsx

```diff
@@ -149,34 +159,54 @@ export const CameraButton = (props: CameraButtonProps): JSX.Element => {
     strings.tooltipDisabledContent = strings.tooltipVideoLoadingContent;
   }
 
+  const cameraOn = props.checked;
+
+  const toggleAnnouncerString = useCallback(
+    (isCameraOn: boolean) => {
+      setAnnouncerString(
+        announcerString === (strings.cameraActionTurnedOffAnnouncement || undefined) || !isCameraOn
```

Comment:
do you need the `(strings.cameraActionTurnedOffAnnouncement || undefined)` in the check?

If announcerString currently equals `cameraActionTurnedOffAnnouncement` and the camera is turnedOff this if will equate to true and announce the camera is turned on